devtools/server/actors/replay/connection.js

```diff
@@ -216,6 +216,8 @@ class Recording extends EventEmitter {
     }
 
     try {
+      this.emit("saved", data);
```

Comment:
I realize actually, can we move this up to after the first `Internal.setRecordingMetadata`?